tools/topology/sof/pipe-amp-ref-capture.m4

```diff
@@ -38,17 +38,26 @@ ifdef(`SMART_REF_PPL_ID',`',`errprint(note: Need to define Echo Ref pipeline ID
 ifdef(`SMART_REF_CH_NUM',`',`errprint(note: Need to define Echo Ref channel number for sof-smart-amplifier
 )')
 
+ifelse(SMART_FB_CHANNELS, `8',
+`define(`FB_CHMAP',`0x01,0x02,0x04,0x08,0x10,0x20,0x40,0x80')',
+`define(`FB_CHMAP',`0x01,0x02,0x04,0x08,0x00,0x00,0x00,0x00')'
+)
+
+ifelse(SMART_REF_CHANNELS, `8',
+`define(`REF_CHMAP',`0x01,0x02,0x04,0x08,0x00,0x00,0x00,0x00')',
```

Comment:
it's fine to keep this for smart_amp_test at the moment, we might need to pick the 4 'V's only for 4 speakers in the future.